OneBlink.SDK.Tests/KeysClientTests.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.IO;
+using System.Net;
+using dotenv.net;
+using OneBlink.SDK;
+using OneBlink.SDK.Model;
+using Xunit;
+
+namespace OneBlink.SDK.Tests
+{
+  public class KeysClientTests
+  {
+    private string ACCESS_KEY;
+    private string SECRET_KEY;
+    private string apiOrigin;
```

Comment:
Can we remove this if its not being used?

Reply:
yep